SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -1775,6 +1775,7 @@ int vtkSlicerSkeletalRepresentationInitializerLogic::TransformPoints(itkThinPlat
     writer->SetFileName(outputFileName.c_str());
     writer->SetInputData(poly);
     writer->Update();
+    return 0;
```

Comment:
If the return is always 0, make the function void instead of int and remove the return.